eth_account/signers/local.py

```diff
@@ -46,7 +46,7 @@ def address(self):
     @property
     def privateKey(self):
         """
-        .. CAUTION:: Deprecated for :var:`~eth_account.signers.local.LocalAccount.key`.
+        .. CAUTION:: Deprecated for :meth:`~eth_account.signers.local.LocalAccount.key`.
```

Comment:
Ah interesting, so the link changes to `:meth:` when it's a `@property`. Did this show up during doc tests? You were able to confirm that it links correctly with this change?

Reply:
Yep, it was showing up as a warning in the doc build. And it does work correctly!